src/components/NavPage.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import PropTypes from "prop-types";
+import { AppContext } from "../contexts/AppContext";
+
+function NavPage(props) {
+  const { id, children } = props;
+  const { activeNavPage } = useContext(AppContext);
+  return (
+    <div
+      tabIndex="0"
+      id={`aic-ct-nav-page-${id}`}
+      aria-labelledby={`aic-ct-nav-button-${id}`}
+      aria-hidden={activeNavPage !== id}
+      style={activeNavPage !== id ? { display: "none" } : {}}
+    >
```

Comment:
Look into using hidden attribute

Reply:
Had a look at this, after reading [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) and [CSSTricks](https://css-tricks.com/the-hidden-attribute-is-visibly-weak/) I feel like I understand this a little better, and it's a nice trick... But that I don't feel it's worth changing in this case

Reasons being:
- I suspect we'll land on different CSS later
- Being easily overrideable It seems possible `hidden` may show the element when we don't expect it to
- Not transitionable (neither is `display:none`, but using different CSS is less of a leap than not using any CSS)